releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml

```
@@ -0,0 +1,52 @@
+---
+prelude: >
+    The Qiskit 1.1.0 release is a minor feature release that includes a myriad of new feature and bugfixes.
+    The highlights for this release are:
+
+      * Support for typed classical variables has been added to Qiskit's :class:`.QuantumCircuit`. These
+        classical variables can be specified as inputs or as scoped variables in a :class:`.QuantumCircuit`
+        where they e.g. store the output of qubit measurements or target control-flow operations. Support
+        for e.g. setting gate parameters or output variables will be added in the future.
+
+      * The default two qubit synthesis methods that are used internally by the transpiler in the
+        :class:`.UnitarySynthesis` pass have been re-implemented in Rust. This yields significant
+        runtime speedups when decomposing two qubit unitary matrices. As a consequence, the runtime of
+        transpilation with optimization level 3 was significantly improved where running
+        :class:`.UnitarySynthesis` incurred a large runtime overhead historically. This release also
+        starts running :class:`.UnitarySynthesis` as part of the optimization stage in optimization level
+        2 because of these runtime performance improvements.
+
+        Additionally, the numeric methods used in :class:`.Isometry` have been moved to rust,
```

some other instances of the name are capitalized.

```suggestion
        Additionally, the numeric methods used in :class:`.Isometry` have been moved to Rust,
```